### DISCUSSION TITLE
Multiple assignees on Issues and Pull requests

### DISCUSSION THREAD

**Discussion by Y-O-W** [community/community#44078] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

According to this blog post the feature should be live: https://github.blog/2016-05-27-multiple-assignees-on-issues-and-pull-requests/. But it's not working properly for our organization and repositories.

**Comment by MarcHagen** [comment]
Works fine, can multi assign on issues and PRs. Blog post from 2016?

**Reply by Y-O-W** [reply]
As far as I can tell, it's a pro feature. At least for PRs.

**Reply by MarcHagen** [reply]
https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users

> Who can use this feature
> Anyone with write access to a repository can assign issues and pull requests.

You can assign multiple people to each issue or pull request, including yourself, anyone who has commented on the issue or pull request, anyone with write permissions to the repository, and organization members with read permissions to the repository. For more information, see "[Access permissions on GitHub](https://docs.github.com/en/articles/access-permissions-on-github)."

**Comment by yangwao** [comment]
Hey seems it stopped working?
